action.yaml

```diff
@@ -1,14 +1,18 @@
 name: Dependabot composer diff
 description: Posts a comment with a diff when Composer dependencies are updated
 inputs:
-  token:
+  path_prefix:
```

Comment:
I'd rather just call this `path`, since it's not just a prefix: it's the entire path.

Reply:
I actually made that very change and reverted it. Because then it'd make sense to use `$PATH` as environment variable, which might not be the best thing to do :thinking:

Also, https://github.com/davidrjonas/composer-lock-diff#options mentions it being used as a prefix. Perhaps `base_path` and `$BASE_PATH`?

Reply:
let's go with `base_path` 👍